e2e/testcafe-devextreme/tests/treeList/markup.ts

```diff
@@ -159,3 +159,62 @@ const tasksT1223168 = [{
     await changeTheme(Themes.genericLight);
   });
 });
+
+// T1291705
+test('The shading should alternate correctly after expanding the node when repaintChangesOnly is enabled', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const treeList = new TreeList('#container');
+
+  await treeList.apiExpandRow(4);
+  await treeList.apiExpandRow(2);
+
+  await takeScreenshot('row_alternation_after_expanding_node_when_repaintChangesOnly_=_true', treeList.element);
```

Comment:
Can we rename screenshot filename to: "T1291705-row-alternation-after-expanding-node-when-row_alternation_after_expanding_node_when_repaintChangesOnly=true" or something like this, because filenames in the etalons folder use kebab case